src/core/toga/widgets/button.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, label, id=None, style=None, on_press=None, enabled=True, fact
         # Set all the properties
         self.label = label
         self.on_press = on_press
+        self.enabled = enabled
```

Comment:
This *shouldn't* be required here - enabled/not enabled should be stored at the Widget level.